package com.invoiceGeneration.config;

import io.swagger.v3.oas.models.OpenAPI;
import io.swagger.v3.oas.models.info.Info;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class OpenAPIConfig {

    @Bean
    public OpenAPI customOPOpenAPI(){
        return new OpenAPI()
                .info( new Info()
                        .title("Invoice Generator Micro Saas")
                        .version("1.0.0.1")
                        .description("API documentation for Invoice System")
                );
    }
}
